Tests/Fluent.Tests/WebApp/WebAppsTests.cs

```diff
@@ -156,5 +156,43 @@ public void WebAppWithoutPhp()
                 }
             }
         }
+
+        [Fact]
+        public void CanCRUDWebAppWithContainer()
+        {
+            using (var context = FluentMockContext.Start(this.GetType().FullName))
+            {
+                string GroupName1 = TestUtilities.GenerateName("javacsmrg");
+                string WebAppName1 = TestUtilities.GenerateName("java-webapp-");
+                string AppServicePlanName1 = TestUtilities.GenerateName("java-asp-");
+
+                var appServiceManager = TestHelper.CreateAppServiceManager();
+
+                try
+                {
+                    var appServicePlan = appServiceManager.AppServicePlans
+                        .Define(AppServicePlanName1)
+                        .WithRegion(Region.USEast)
+                        .WithNewResourceGroup(GroupName1)
+                        .WithPricingTier(PricingTier.PremiumP1v3)
+                        .WithOperatingSystem(OperatingSystem.Windows)
+                        .Create();
+
+                    var webApp = appServiceManager.WebApps.Define(WebAppName1)
+                        .WithExistingWindowsPlan(appServicePlan)
+                        .WithExistingResourceGroup(GroupName1)
+                        .WithPublicDockerHubImage("mcr.microsoft.com/azure-app-service/samples/aspnethelloworld:latest")
+                        .Create();
```

Comment:
Serve as sample as well.

From Portal, web app with windows docker seems had to use PremiumP1v3 or higher.